TestAssets/TestProjects/MSBuildIntegration/build.proj

```diff
@@ -35,11 +35,11 @@
     <Error Condition=" '$(MSBuildSDKsPath)' == '' " 
            Text="Expected MSBuildSDKsPath to be set, but it is not." />
 
-    <Error Condition=" !Exists('$(MSBuildSDKsPath)/%(Sdk.Identity)/%(Sdk.Version)/build/InitialImport.props') " 
-           Text="Expected '$(MSBuildSDKsPath)/%(Sdk.Identity)/%(Sdk.Version)/build/InitialImport.props' to exist, but it does not. " />
+    <Error Condition=" !Exists('$(MSBuildSDKsPath)/%(Sdk.Identity)/%(Sdk.Version)/sdk/Sdk.props') " 
+           Text="Expected '$(MSBuildSDKsPath)/%(Sdk.Identity)/%(Sdk.Version)/build/Sdk.props' to exist, but it does not. " />
 
-    <Error Condition=" !Exists('$(MSBuildSDKsPath)/%(Sdk.Identity)/%(Sdk.Version)/build/FinalImport.targets') " 
-           Text="Expected '$(MSBuildSDKsPath)/%(Sdk.Identity)/%(Sdk.Version)/build/FinalImport.targets' to exist, but it does not. " />
+    <Error Condition=" !Exists('$(MSBuildSDKsPath)/%(Sdk.Identity)/%(Sdk.Version)/sdk/Sdk.targets') " 
+           Text="Expected '$(MSBuildSDKsPath)/%(Sdk.Identity)/%(Sdk.Version)/build/Sdk.targets' to exist, but it does not. " />
```

Comment:
Same here as previous, Sdk should be capitalized and build should be changed to Sdk in the error text.